estate/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id/id,group_id/id,perm_read,perm_write,perm_create,perm_unlink
+access_estate_property,access_estate_property,model_estate_property,base.group_user,1,1,1,1
```

Comment:
```suggestion
access_estate_property,access.estate.property,model_estate_property,base.group_user,1,1,1,1
```
The name of a model should always be separated by dots, not commas

Reply:
In chapter 5 of the Security, there were instruction on replacing the . with _ 
![image](https://github.com/odoo/tutorials/assets/160074741/06cf39a0-39ad-4149-a7c7-8de9a9206cd7)

Reply:
Or is the case for only model_id/id column? I am a bit confused tbh

Reply:
Yes the underscore that replaces the dot is the case for only model_id/id. It's true that this part of the documentation might not be very clear. Otherwise the best doc is always the code, don't hesitate to open modules in Odoo to see how it's done